src/main/java/com/internet/shop/controllers/AddProductAdminController.java

```diff
@@ -0,0 +1,41 @@
+package com.internet.shop.controllers;
+
+import com.internet.shop.lib.Injector;
+import com.internet.shop.model.Product;
+import com.internet.shop.service.ProductService;
+import java.io.IOException;
+import javax.servlet.ServletException;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+public class AddProductAdminController extends HttpServlet {
```

Comment:
We can call this class as AddProductController

Reply:
done